Added miller_rabin.py and inversion_count.py

anirudnits opened the pull request:
Added the _miller_rabin.py_ file in **/Algorithms/Number-Theory/Primality-Tests** directory and _inversion_count.py_ in **/Algorithms/Other** directory.

KruZZy commented:
Very good job! However, for the Inversion Count problem, I think that you should add some explanations for this approach. I haven't seen a different approach than the modified MergeSort and Binary Indexed Trees personally and I am probably not the only one who would like to understand how your method works.

anirudnits commented:
@KruZZy made some changes to the inversion_count,py file. Earlier I was using a simple list considering amortized insertion time for insertion(which worked for me in AtCoder contests) but now changed it to a SortedList to get a O(logn) insertion time in worst case and also added the explanation for the same.
